src/containerapp/azext_containerapp/_utils.py

```diff
@@ -436,10 +437,11 @@ def _get_oryx_run_image_version_key(tag_obj, framework):
 def parse_oryx_run_image(image_repository, tag) -> OryxRunImageTagProperty:
     # Example Oryx run image: mcr.microsoft.com/oryx/<platform>:<version>-<os>
     # Note: image_repository should NOT include the "mcr.microsoft.com/" prefix; it should ONLY be the repository
-    re_matches = re.findall(r"oryx\/([A-Za-z]*):([0-9.]*)-([A-Za-z-]*)", "{}:{}".format(image_repository, tag))
-    if len(re_matches) == 0:
+    re_matches = re.findall(r"(oryx|azure-buildpacks)/([A-Za-z]*):([0-9.]*)-([A-Za-z-]*)",
```

Comment:
I think the RegEx is broken here: after the `(oryx|azure-buildpacks)` term, the forward slash should be escaped, `\/`

I just tested this RegEx offline and it wasn't matching sample repo-tag combinations unless the backslash was added.

Full RegEx in case you want to copy-paste:

```
(oryx|azure-buildpacks)\/([A-Za-z]*):([0-9.]*)-([A-Za-z-]*)
```

Reply:
added escape for  /